curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -152,6 +152,8 @@ int InitLog(const char *confPath, const char *argv0) {
     FLAGS_vlog_level = FLAGS_v;
 
     // initialize logging module
+    // NOTE: https://github.com/google/glog#setting-flags
+    FLAGS_stderrthreshold = 3;
```

Comment:
If more than one module needs this, it's better to extract it as a function, like,
`DisableLoggingToStdErr`, and you can put this function under src/common folder.

Reply:
I will make a force push latter.

Reply:
Done.